File: apiEZPZ/src/main/java/com/example/apiezpz/auth/security/JwtProperties.java
package com.example.apiezpz.auth.security;

import lombok.Getter;
import lombok.Setter;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.stereotype.Component;

@ConfigurationProperties(prefix = "jwt")
@Component
@Getter
@Setter
public class JwtProperties {
    private String secret;
    private long accessTokenExpirationMinutes;
    private long refreshTokenExpirationMinutes;
}

File: backend/src/main/kotlin/com/example/apiezpz/auth/security/JwtProperties.kt
package com.example.apiezpz.auth.security

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.stereotype.Component

@ConfigurationProperties(prefix = "jwt")
@Component
data class JwtProperties(
    var secret: String = "",
    var accessTokenExpirationMinutes: Long = 0,
    var refreshTokenExpirationMinutes: Long = 0
) 